README.md

```diff
@@ -404,6 +404,7 @@ npm install -g @angular/cli@latest
 Local project package:
 ```bash
 rm -rf node_modules dist # use rmdir on Windows
+npm uninstall --save-dev angular-cli
```

Comment:
But lines 399-401 are about cleaning and installing "global package" and this one is about the "local project package" .